### PR TITLE
feat: offer-design-system playground 구현

### PR DESCRIPTION
**Pull request by sonsurim**
## ⛳ 구현 사항

- 간혹 storybook 환경과 웹에 차이가 발생하여 storybook이 아닌 웹에서 ods를 테스트해볼 수 있게 playground를 구현하였습니다.

### CODE REVIEW

File: apps/ods-playground/src/index.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
+:root {
+  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
+  line-height: 1.5;
+  font-weight: 400;
```
[the remark this answers is not in the document]
Comment (sonsurim) [reply apps/ods-playground/src/index.css]: [chore: remove unnecessary files and change the title](https://github.com/price-offer/offer-design-system/pull/205/commits/d378dcab6444a6bdafc462a65c0cdcdb1c72e9b8)에서 반영 하였습니둥!

File: apps/ods-playground/src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { Avatar } from '@offer-ui/react'
+
+function App() {
+
+  return (
+    <Avatar alt="avatar" src="https://picsum.photos/100"/>
```
[the remark this answers is not in the document]
Comment (sonsurim) [reply apps/ods-playground/src/App.tsx]: [chore: remove unnecessary files and change the title](https://github.com/price-offer/offer-design-system/pull/205/commits/d378dcab6444a6bdafc462a65c0cdcdb1c72e9b8)에서 반영하였습니둥!

File: apps/ods-playground/index.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + React + TS</title>
```
[the remark this answers is not in the document]
Comment (sonsurim) [reply apps/ods-playground/index.html]: [chore: remove unnecessary files and change the title](https://github.com/price-offer/offer-design-system/pull/205/commits/d378dcab6444a6bdafc462a65c0cdcdb1c72e9b8)에서 반영하였습니둥!

File: apps/ods-playground/src/App.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (sonsurim) [reply apps/ods-playground/src/App.tsx]: [docs: add ods playground README](https://github.com/price-offer/offer-design-system/pull/205/commits/08c6b98fdcdbf30daa276d11988a8aa2d78e8dc2)

요기서 README도 업데이트 했심니다!

**Review by shinhyojeong — APPROVED** [review]
영어 좀 치시네예 👍

**Pull request merged**